README.md

```diff
@@ -0,0 +1,57 @@
+# Today Migration Plugin #
+
+Provides a series of wp cli tasks for manipulating data used by the old Today-Bootstrap theme to work with the new theme and associated plugins.
+
+
+## Description ##
+
+Provides a series of wp cli tasks for manipulating data used by the old Today-Bootstrap theme to work with the new theme and associated plugins.
+
+
+## Documentation ##
+
+Head over to the [Today Migration Plugin wiki](https://github.com/UCF/Today-Migration-Plugin/wiki) for detailed information about this plugin, installation instructions, and more.
```

Comment:
Remove note about documentation here (I imagine we won't be setting up a wiki for this since this is a one-time-use plugin)

Reply:
Removed. 🎊